src/Distributed/distributed_fft_based_poisson_solver.jl

```diff
@@ -38,32 +38,35 @@ infer_transform(::Flat) = PencilFFTs.Transforms.NoTransform!()
 Return a solver for the Poisson equation,
 
 ```math
-∇²x = b
+∇²φ = b
```

Comment:
We call it `x` in `solve!` -- should we change it there too? Trying to use common notation all over...

Reply:
Hmm I guess this is ok. We can't read it if we use `x` here because we also use `x` for the coordinate.